### PR TITLE
fix(tracing): fix log4js message format call

### PR DESCRIPTION
**Pull request by basti1302**
refs 103123

### CODE REVIEW

File: packages/collector/test/tracing/logger/log4js/app.js [hunk 1]
```diff
@@ -57,7 +58,11 @@ app.post('/log', (req, res) => {
     method = logger[level];
   }
 
-  args.push(message);
+  if (multipleArguments) {
+    args.push(message, 'more', 'arguments');
```
Comment (basti1302) [inline packages/collector/test/tracing/logger/log4js/app.js:62]: This was enough to reproduce the `"TypeError: CreateListFromArrayLike called on non-object"` that has been mentioned in the original ticket.

File: packages/collector/test/tracing/logger/log4js/test.js [hunk 2]
```diff
@@ -38,23 +44,22 @@ mochaSuiteFn('tracing/logger/log4js', function () {
...
+          span => expect(span.f.e).to.equal(String(controls.getPid())),
+          span => expect(span.f.h).to.equal('agent-stub-uuid')
+        ]);
+        expectAtLeastOneMatching(spans, span => {
+          checkNextExitSpan(span, entrySpan);
+        });
+        const log4jsSpans = getSpansByName(spans, 'log.log4js');
+        expect(log4jsSpans).to.be.empty;
+      });
+    });
```
Comment (basti1302) [inline packages/collector/test/tracing/logger/log4js/test.js:62]: Mostly refactored some `.then` to async/await style in the test, that's why the diff is a bit larger than absolutely necessary for the actual fix.

File: packages/collector/test/tracing/logger/log4js/test.js [hunk 3]
```diff
@@ -63,44 +68,65 @@ mochaSuiteFn('tracing/logger/log4js', function () {
...
+
+    it(`must trace fatal ${suffix}`, () =>
+      runTest({
+        level: 'fatal',
+        message: 'Fatal message - should be traced.',
+        useLogMethod,
+        expectErroneous: true
+      }));
+
+    it(`must trace error ${suffix} with multiple log arguments`, () =>
```
Comment (basti1302) [inline packages/collector/test/tracing/logger/log4js/test.js:101]: this one is new

File: packages/core/src/tracing/instrumentation/loggers/log4js.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ function instrumentedLog(ctx, originalArgs, originalLog, markAsError) {
     } else {
       // The original log4js log method takes (level, ...data) as arguments and creates the actually logged message via
       // util.format(...data).
-      message = util.format.apply(...originalArgs.slice(1));
+      message = util.format(...originalArgs.slice(1));
```
Comment (basti1302) [inline packages/core/src/tracing/instrumentation/loggers/log4js.js:65]: Aaand this is the actual fix.

**Review by kirrg001 — APPROVED** [review]
👍

**Pull request merged**